server/log_rpc_server_test.go

```diff
@@ -9,22 +9,29 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/golang/protobuf/proto"
 	"github.com/google/trillian"
+	"github.com/google/trillian/crypto"
+	"github.com/google/trillian/merkle"
 	"github.com/google/trillian/storage"
 	"github.com/google/trillian/testonly"
 	"golang.org/x/net/context"
 )
 
+var th = merkle.NewRFC6962TreeHasher(crypto.NewSHA256())
+
 var logID1 = int64(1)
 var logID2 = int64(2)
 var leaf0Request = trillian.GetLeavesByIndexRequest{LogId: logID1, LeafIndex: []int64{0}}
 var leaf0Minus2Request = trillian.GetLeavesByIndexRequest{LogId: logID1, LeafIndex: []int64{0, -2}}
 var leaf03Request = trillian.GetLeavesByIndexRequest{LogId: logID1, LeafIndex: []int64{0, 3}}
 var leaf0Log2Request = trillian.GetLeavesByIndexRequest{LogId: logID2, LeafIndex: []int64{0}}
 
-var leaf1 = trillian.LogLeaf{LeafIndex: 1, MerkleLeafHash: []byte("hash"), LeafValue: []byte("value"), ExtraData: []byte("extra")}
-var leaf3 = trillian.LogLeaf{LeafIndex: 3, MerkleLeafHash: []byte("hash3"), LeafValue: []byte("value3"), ExtraData: []byte("extra3")}
-var expectedLeaf1 = trillian.LogLeaf{LeafIndex: 1, MerkleLeafHash: []byte("hash"), LeafValue: []byte("value"), ExtraData: []byte("extra")}
-var expectedLeaf3 = trillian.LogLeaf{LeafIndex: 3, MerkleLeafHash: []byte("hash3"), LeafValue: []byte("value3"), ExtraData: []byte("extra3")}
+var leaf1Data = []byte("value")
+var leaf3Data = []byte("value3")
+
+var leaf1 = trillian.LogLeaf{LeafIndex: 1, MerkleLeafHash: th.HashLeaf(leaf1Data), LeafValue: leaf1Data, ExtraData: []byte("extra")}
+var leaf3 = trillian.LogLeaf{LeafIndex: 3, MerkleLeafHash: th.HashLeaf(leaf3Data), LeafValue: leaf3Data, ExtraData: []byte("extra3")}
+var expectedLeaf1 = trillian.LogLeaf{LeafIndex: 1, MerkleLeafHash: th.HashLeaf(leaf1Data), LeafValue: leaf1Data, ExtraData: []byte("extra")}
```

Comment:
Is there any difference between leaf1 and expectedLeaf1? Could we do "expectedLeaf1 = leaf1"?

If not, could we do:
var expectedLeaf1 = trillian.LogLeaf{LeafIndex: leaf1.LeafIndex, MerkleLeafHash: leaf1.MerkleLeafHash, LeafValue: leaf1.LeafValue, ExtraData: leaf1.ExtraData}
?

(Ditto for leaf3 / expectedLeaf3.)